src/Goteo/Model/Project.php

```diff
@@ -33,14 +33,15 @@ class Project extends Model {
     use Traits\SdgRelationsTrait;
 
     // PROJECT STATUS IDs
+    const STATUS_NEGOTIATION = -2;
```

Comment:
Improvement from review face to face:

 // STATUS TO BE USED TO FILTER PROJECT
    const STATUS_DRAFT       = -1; // is this really necessary?
    const STATUS_NEGOTIATION = -2; // filter status, project's real status is self::STATUS_EDITING

    // PROJECT STATUS IDs
    const STATUS_REJECTED    = 0;
    const STATUS_EDITING     = 1; // editing or negotiating (if draft)
    const STATUS_REVIEWING   = 2;
    const STATUS_IN_CAMPAIGN = 3;
    const STATUS_FUNDED      = 4;
    const STATUS_FULFILLED   = 5;
    const STATUS_UNFUNDED    = 6;